### ISSUE TITLE
Does SCIP.jl support branch and price?

### ISSUE THREAD

**Issue by Darksonn** [scipopt/SCIP.jl#181]
I know that I can use SCIP as a branch-and-price framework, but is unclear whether this wrapper supports those parts of SCIP.

**Comment by rschwarz** [comment]
No, the pricer plugin infrastructure is not wrapped, currently.

**Comment by matbesancon** [comment]
Isn't there the raw wrapper? https://github.com/scipopt/SCIP.jl/blob/master/src/wrapper/pub_pricer.jl

Note: this means you would have to use it in a fairly raw interface with managed-scip instead of the MOI wrapper

**Comment by rschwarz** [comment]
True, you can do anything manually, but there is no specific support through code in SCIP.jl.

To be honest, I don't know enough about what would be requried in terms of memory management etc.

**Comment by odow** [comment]
Closing as won't fix. We have no plans to add this to the MOI wrapper or make a higher level Julia interface. The C wrapper is there if you really need it.
